.travis.yml

```diff
@@ -4,10 +4,11 @@
 #
 # For more information, see https://github.com/haskell-CI/haskell-ci
 #
-# version: 0.3.20190425
+# version: 0.3.20190804.2
 #
 language: c
-dist: xenial
+dist: bionic
+sudo: required
```

Comment:
I'm very surprised that `sudo: required` is necessary now. Doesn't that prevent Travis from using [container-based builds](https://blog.travis-ci.com/2014-12-17-faster-builds-with-container-based-infrastructure/), which are purported to start faster than other builds?

Reply:
It’s required as I cannot add ghcjs repository otherwise. VM images are slower to start, but OTOH are beefier machines, maybe it evens out.